cupyx/scipy/sparse/linalg/_iterative.py

```diff
@@ -33,6 +33,8 @@ def cg(A, b, x0=None, tol=1e-5, maxiter=None, M=None, callback=None,
             iteration. It is called as ``callback(xk)``, where ``xk`` is the
             current solution vector.
         atol (float): Tolerance for convergence.
+        residual_interval (int): Iterations after which residual must be
+            recomputed.
```

Comment:
Instead of adding a parameter that doesn't exist in the scipy signature of this function, maybe you can try to replicate the behavior and calculate the value at the same time scipy does.